apps/web/src/styles/index.css

```diff
@@ -1,7 +1,7 @@
 :root {
     max-width: 1280px;
```

Comment:
remove this

Reply:
Added and Pushed to commit 93ef13d6c7a0956d3c3d1310b60d85fafa8b09c5.